example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,43 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(collection, ['createdAt', 'updatedAt']));
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      const updatedCollection = {
+        ...collection,
+        reportToEms: false,
+      };
+      await updateCollection({
+        prefix: config.stackName,
+        collection: updatedCollection,
+      });
+
+      await expectAsync(waitForObjectToExist({
+        bucket: config.bucket,
+        key: recordUpdatedKey,
+      })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordUpdatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Update');
+      expect(omit(message.record, ['createdAt', 'updatedAt', 'files'])).toEqual(omit(updatedCollection, ['createdAt', 'updatedAt', 'files']));
```

Comment:
Why are we omitting `files`?

Reply:
Ah sorry that was residual that I didn't clean up from my first round of implementing this test

Reply:
[`8d76b1c` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/8d76b1c48461cee69ebff33bd04b3d57dc3c0796)